### PR TITLE
struggling with travis

### PR DESCRIPTION
**Pull request by gcolvin**
another try at getting this past travis

### CODE REVIEW

File: EIPS/eip-2315.md [hunk 1]
```diff
@@ -145,24 +143,12 @@ Bytecode: `0x600556b2b75b6003b3`
...
-
-|  Pc   |      Op     | Cost |   Stack   |   RStack  |
-|-------|-------------|------|-----------|-----------|
-|    0  |   BEGINSUB  |    1 |        [] |        [] |
-
-
-```
-Error: at pc=0, op=BEGINSUB: invalid subroutine entry
-```
+Consumed gas: `26`
```
Comment (holiman) [inline EIPS/eip-2315.md:151]: @gcolvin Why did you revert this? I had updated these testcases

**Comment by holiman** [comment]
@gcolvin please make sure you make your changes based off the most recent `master` version. You just removed all the updated testcases I had fixed yesterday, plus some clarifications from https://github.com/ethereum/EIPs/pull/2656

**Comment by matkt** [comment]
When you say "JUMPSUB be low, and RETURNSUB be verylow" Do you mean JUMPSUB = 5 and RETURNSUB = 3? Because I see that you have corrected JUMPSUB but RETURNSUB is equal to 2 on the tests

**Comment by holiman** [comment]
This PR made some random changes here and there, but the PR description does not even mention what the _intended_ changes were -- I don't know which ones were intentional and which ones were somehow just accidents during merge :/

**Comment by holiman** [comment]
From the YP:
```
Gbase 2    Amount of gas to pay for operations of the setWbase.
Gverylow 3    Amount of gas to pay for operations of the setWverylow.
Glow 5    Amount of gas to pay for operations of the setWlow.
Gmid 8    Amount of gas to pay for operations of the setWmid.
```
So jumpsub should be `5`, according to that sentence. IMO it should be at `MID=8`, since that's what a `JUMP` is at. A `JUMPSUB` is _more_ resource intensive than a `JUMP` since it requires an internal stack push, which `JUMP` doesn't.

**Comment by matkt** [comment]
> From the YP:
> 
> ```
> Gbase 2    Amount of gas to pay for operations of the setWbase.
> Gverylow 3    Amount of gas to pay for operations of the setWverylow.
> Glow 5    Amount of gas to pay for operations of the setWlow.
> Gmid 8    Amount of gas to pay for operations of the setWmid.
> ```
> 
> So jumpsub should be `5`, according to that sentence. IMO it should be at `MID=8`, since that's what a `JUMP` is at. A `JUMPSUB` is _more_ resource intensive than a `JUMP` since it requires an internal stack push, which `JUMP` doesn't.

That makes sense

**Pull request merged**